sky/dist/BUILD.gn

```diff
@@ -7,6 +7,7 @@ root_dist_dir = "$root_build_dir/dist"
 copy("sky_viewer") {
   sources = [
     "$root_build_dir/sky_viewer.mojo",
+    "$root_build_dir/symbols/libsky_viewer_library.so",
```

Comment:
Also this tries to copy a libsky_viewer_library on platforms that don't produce a sky_viewer.mojo. this step should only run on linux/android, not mac or ios